### PR TITLE
awesome books code

### PR DESCRIPTION
**Pull request by 29td**
Add script and index.html

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  Storage.addBook(book);
+  UI.clearFields();
+});
+
+// Event: Deletes a Book
+document.querySelector('.book-collection').addEventListener('click', (e) => {
+  UI.deleteBook(e.target);
+  Storage.removeBook(e.target.parentElement.previousElementSibling.textContent);
+});
+
```
Comment (dasileker) [inline script.js:82]: - [ ] Nice job on the app, but the removal is not working I do see it delete but after refreshing the page it returns. hint targeting the id in the local storage

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @29td,** 

### **Good job so far!**
There are some issues that you still need to work on to go to the next project but you are almost there!

## **Highlights**
- linters are passing ✅.
- Readme file is good ✅.
- Commits are well structured ✅.

## Required Changes ♻️
_Check the comments under the review._

- Please make sure to add details to PR, it will be more convenient for the CR to read about the steps you did while coding.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Carshy** [comment]
Changes made as requested and now the program can remove a book when deleted. @dasileker thank you for reviewing our code it was very helpful

**Review by Kandy-Peter — APPROVED** [review]
Hi @team,

### APPROVED 🟢 

Great implementation of all the requirements in this milestone. 👏
Your web app looks really fantastic. 🎉 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

##  Optional suggestion
 
# CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**